.github/workflows/tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9.0-rc - 3.9, pypy2, pypy3]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9.0-rc - 3.9, pypy3]
```

Comment:
Didn't we say we start support with 3.6? To get more of the new features (`typing`, f-strings)

Reply:
I wasn't sure, so I left it for now.

I think 3.6+ is probably the right idea, but I wonder if there's anyone still stuck on 3.5.

Reply:
https://endoflife.date/python. I'd recommend starting with 3.6 (it was my personal choice for most of my projects, no regrets)

Reply:
(Also, according to the [stats](https://pypistats.org/packages/lark-parser), python3.5 is used far less than 2.7)